freeswitch_click2dial/static/src/js/freeswitch_click2dial.js

```diff
@@ -0,0 +1,96 @@
+/*  © 2014-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+    © 2015-2016 Juris Malinens (port to v9)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).   */
+
+odoo.define('freeswitch_click2dial.click2dial', function (require) {
+"use strict";
+
+var UserMenu = require('web.UserMenu');
+var WebClient = require('web.WebClient');
+var web_client = require('web.web_client');
+var Widget = require('web.Widget');
+var core = require('web.core');
+var _t = core._t;
+
+var click2dial = {};
+
+click2dial.OpenCaller = Widget.extend({
+    template: 'freeswitch_click2dial.OpenCaller',
+
+    start: function () {
+        this.$('#freeswitch-open-caller').on(
+            'click', this.on_open_caller);
+        this._super();
+    },
+
+    on_open_caller: function (event) {
+        event.stopPropagation();
+        var self = this;
+        self.rpc('/freeswitch_click2dial/get_record_from_my_channel', {}).done(function(r) {
+        // console.log('RESULT RPC r='+r);
```

Comment:
Sorry again for my lack of precision:
Can you remove dead code from the modules (any commented code in python and jscript modules)?